src/components/dls/KeyboardInput/KeyboardInput.module.scss

```diff
@@ -1,9 +1,15 @@
 @use "src/styles/breakpoints";
+@use "src/styles/theme";
 
 .container {
   direction: ltr;
   color: var(--color-text-default);
-  border: 1px solid var(--color-borders-hairline);
+  @include theme.light {
+    border: 1px solid var(--color-borders-hairline);
```

Comment:
Both light and dark seem to use the same value `border: 1px solid var(--color-borders-hairline);`